src/main/java/com/exasol/sql/expression/rendering/AbstractExpressionRenderer.java

```diff
@@ -1,9 +1,6 @@
 package com.exasol.sql.expression.rendering;
 
-import java.util.ArrayDeque;
-import java.util.Deque;
-
-import com.exasol.sql.expression.*;
+import com.exasol.sql.expression.literal.BooleanLiteral;
```

Comment:
The boolean literal is the only literal to appear in the import. That smells like an inconsistency. Please check.